src/main/java/mate/jdbc/model/Manufacturer.java

```diff
@@ -0,0 +1,45 @@
+package mate.jdbc.model;
+
+public class Manufacturer {
```

Comment:
It's time for big change! 😃 💪 

Many problems in code are caused by allowing too much. We can improve our code by introducing some restrictions to the Manufacturer class that will force it to be used correctly.

1. Make it immutable:
- add `final` to fields
- delete all setters
2. Introduce static factory methods and hide constructor - this way you can determine the intentions behind the creation of objects and ensure that all required fields are completed. We can will need such methods:
```java
    //Used for new Manufacturer
    public static Manufacturer createNew(String name, String country) {
        return new Manufacturer(null, name, country);
    }

    //Used for existing Manufacturer
    public static Manufacturer createExisting(ResultSet resultSet) {
        //retrieve id, name and country from resultSet
        return new Manufacturer(id, name, country);
    }

``` 
3. Introduce copying method for setting ID to new manufacturer
```java
    //Used to assign an ID to new manufacturers
    public Manufacturer withNewId(Long id) {
        return new Manufacturer(id, this.name, this.country);
    }
``` 
4. I propose to override `toString` method for nice printing 💡 

see: https://ahdak.github.io/blog/effective-java-part-1/